extensions/grapher/src/com/google/inject/grapher/AbstractInjectorGrapher.java

```diff
@@ -162,10 +180,28 @@ private void createEdges(Iterable<Edge> edges, Map<NodeId, NodeId> aliases) thro
     }
   }
 
+  private void appendSubs(Iterable<Subgraph> subs, Map<NodeId, NodeId> aliases) {
+    for(Subgraph sub: subs) {
```

Comment:
throughout the file please try to adhere to the google java style (see https://google-styleguide.googlecode.com/svn/trunk/javaguide.html) -- specifically, put a space after for/while and before the `(`.  so:  `for (Subgraph subgraph : subgraphs) {`